api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -60,6 +60,108 @@ The group cell's value with applied [format](/api-reference/_hidden/dxDataGridCo
 The group cell's value as it is specified in a data source.
 
 ---
+
+The format for group cells is `x: y` followed by an optional summary. `x` is the caption and `y` is the value of the column used to group data. In the following example, group cells display the column value only:
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function() {
+        $("#{widgetName}Container").dx{WidgetName}({
+            // ...
+            columns: [ 
+                "ID",
+                "City",
+                {
+                    dataField: "Country",
+                    groupIndex: 0,
+                    groupCellTemplate: function(element, options) {
+                        element.text(options.value);
+                    },
+                },
+            // ...
+            ]
+        });
+    });
+
+##### Angular
+    
+    <!--HTML-->
+    <dx-{widget-name} ... >
+        <dxi-column dataField="City"></dxi-column>
+        <dxi-column 
+            dataField="Country" 
+            [groupIndex]="0" 
+            groupCellTemplate="groupCellTemplate">
+        </dxi-column>
+
+        <div *dxTemplate="let data of 'groupCellTemplate'">
+    	    <div>{{data.value}}</div>
+  	    </div>
+    </dx-{widget-name}>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} ... >
+            <DxColumn data-field="City"/>
+            <DxColumn
+                data-field="Country"
+                :group-index="0"
+                group-cell-template="groupCellTemplate"
+            />
+
+            <template #groupCellTemplate="{ data }">
+                <div>{{data.value}}</div>
+            </template>
+        </Dx{WidgetName}>
+    </template>
+    <script>
+    import Dx{WidgetName} from 'devextreme-vue/{widget-name}';
+
+    export default {
+        components: {
+            Dx{WidgetName}
+        },
+        // ...
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import {WidgetName} from 'devextreme-react/{widget-name}';
+
+    class App extends React.Component {
```

Comment:
In new examples, we use components-functions instead of components-classes. [Example](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#template)